src/custom-element.js

```diff
@@ -964,11 +964,13 @@ function createBaseCustomElementClass(win) {
         const ampdoc = ampdocService.getAmpDoc(this);
         this.ampdoc_ = ampdoc;
         // Load the pre-stubbed extension if needed.
-        const extensionId = this.tagName.toLowerCase();
+        const extensionId = this.localName;
         if (!this.implClass_ && !ampdoc.declaresExtension(extensionId)) {
           Services.extensionsFor(win).installExtensionForDoc(
             ampdoc,
-            extensionId
+            extensionId,
+            // The legacy auto-extensions are always 0.1.
```

Comment:
Is this _always_ true? What about https://amp.dev/documentation/components/amp-carousel/?format=websites

Reply:
Correct. This is always true. This only covers a few very early elements that we used to keep inside `v0.js` and then moved into their own extensions. This includes `amp-ad` and `amp-video` and only `0.1` versions. All other versions must be specified explicitly. 

Another clarification: this code has always installed `0.1` version. I'm switching it to explicit value because I'm working on removing `0.1` as the default.

Reply:
Thanks for the clarification!